candidate-standard/openapi/examples/collection.json

```diff
@@ -0,0 +1,646 @@
+                "Air Temperature",
+                "Weather",
+                "Relative Humidity",
+                "Dew point",
+                "Pressure",
+                "Pressure Tendancy",
+                "Visibility"
+            ],
+            "links": [
+                {
+                    "href": "http://www.example.org/uk-hourly-site-specific-observations",
+                    "hreflang": "en",
+                    "rel": "service-doc",
+                    "type": "text/html"
+                },
+                {
+                    "href": "http://www.example.org/terms-and-conditions---datapoint#datalicence",
+                    "hreflang": "en",
+                    "rel": "license",
+                    "type": "text/html"
+                },
+                {
+                    "href": "https://www.metoffice.gov.uk/services/data/datapoint/terms-and-conditions---datapoint#termsofservice",
+                    "hreflang": "en",
+                    "rel": "restrictions",
+                    "type": "text/html"
+                },
+                {
+                    "href": "http://www.example.org/edr/collections/hrly_obs/position?coords={coords}",
+                    "hreflang": "en",
+                    "rel": "data",
+                    "title": "",
+                    "variables": {
+                        "title": "Position query",
+                        "description": "Position query",
+                        "query_type": "position",
+                        "templated": true,
+                        "coords" :{
+                            "description": "Well Known Text POINT value i.e. POINT(-120, 55)"
+                        },
+                        "output_formats": [
+                            "CoverageJSON",
+                            "GeoJSON",
+                            "IWXXM"
+                        ],
+                        "default_output_format": "IWXXM",
+                        "crs_details": [
+                            {
+                                "crs": "CRS84",
+                                "wkt": "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.01745329251994328,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]]"
+                            }
+                        ]    
+                    }
+                },
+                {
+                    "href": "http://www.example.org/edr/collections/hrly_obs/radius?coords={coords}",
+                    "hreflang": "en",
+                    "rel": "data",
+                    "title": "",
+                    "variables": {
+                        "title": "Radius query",
+                        "description": "Radius query",
+                        "query_type": "radius",
+                        "templated": true,
```

Comment:
`"templated": true` should be part of the link object, not the variables object (needs to be updated in other areas of this PR as well).